src/Kentico.Xperience.Siteimprove/Services/ISiteimproveScriptsRenderer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using CMS;
+using CMS.Core;
+
+using Kentico.Xperience.Siteimprove;
+
+using Microsoft.AspNetCore.Html;
+
+[assembly: RegisterImplementation(typeof(ISiteimproveScriptsRenderer), typeof(SiteimproveScriptsRenderer), Lifestyle = Lifestyle.Singleton, Priority = RegistrationPriority.SystemDefault)]
```

Comment:
We should only use this kind of service registration if we have to.

Since we have the built-in ASP.NET Core container available to us, these services should be registered in your `IServiceCollection` extension method.

Reply:
Switched to ASP.NET Core's DI, registration is now part of of `AddSiteimprove` extension method.